{{ cookiecutter.project_shortname }}/.github/workflows/tests.yml

```diff
@@ -21,35 +21,82 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
+          # You can add/remove combinations e.g. dev requirements or postgresql13 by adding
+          # a new item to the following lists.
           python-version: [3.6, 3.7, 3.8, 3.9]
-          # You can add more combinations e.g. dev requirements or MySQL by adding
-          # a new item to this list. Add its corresponding definition below.
-          # EXTRAS and REQUIREMENTS_LEVEL are always needed.
-          services: [release, lowest, devel]
+          requirements-level: [min, pypi]
+          db-service: [postgresql9, postgresql11, mysql5, mysql8]
+          es-service: [es6, es7]
```

Comment:
This will leave us with 64 builds... I added some excludes below.. but I would be nice to have a list of what we need to be tested (cc @lnielsen) so we can compile them into excludes elements (see below).

Reply:
IMHO, we're testing too many Python versions. Maybe we can only keep the edges?  Or edges + most common version?